app/build.gradle

```diff
@@ -79,4 +79,10 @@ dependencies {
     }
     compile 'com.jakewharton:butterknife:6.0.0'
     compile 'com.mikepenz.aboutlibraries:library:4.5.0@aar'
+
+    // rest client for http interaction and dependencies
+    compile 'com.squareup.retrofit:retrofit:1.8.0'
+    compile 'com.squareup.okio:okio:1.0.1'
+    compile 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
+    compile 'com.squareup.okhttp:okhttp:2.0.0'
```

Comment:
Duplicate and out-of-date dependencies:

please replace whole dependency section with:

```
dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile 'com.android.support:appcompat-v7:21.0.3'
    compile 'com.android.support:recyclerview-v7:21.0.3'
    compile 'com.android.support:cardview-v7:21.0.3'
    compile 'com.google.android.gms:play-services:6.5.87'
    compile 'com.squareup.picasso:picasso:2.4.0'
    compile 'io.reactivex:rxandroid:0.24.0'
    compile 'pl.charmas.android:android-reactive-location:0.4@aar'
    compile 'com.github.castorflex.smoothprogressbar:library-circular:1.0.2'
    compile('com.crashlytics.sdk.android:crashlytics:+') {
        transitive = true;
    }
    compile 'com.jakewharton:butterknife:6.0.0'
    compile 'com.mikepenz.aboutlibraries:library:4.5.0@aar'

    // rest client for http interaction and dependencies
    compile 'com.squareup.retrofit:retrofit:1.9.0'
    compile 'com.squareup.okhttp:okhttp:2.2.0'
}
```